asm/code_8085B0C.s

```diff
@@ -450,6 +452,7 @@ _08085E90: .4byte gUnknown_202ECA4
 _08085E94: .4byte 0x000181e4
 	thumb_func_end SetDungeonBGColorRGB
 
+@ gDungeon->field_0x18215 = 0;
```

Comment:
Is this comment intentional, or should it be a decomp.me link like the others? Likewise with `sub_80861A8`.

Reply:
@Kermalis confirmed that these are one-liners.